homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,85 @@
+const tamagochi = {
+    name: 'тамагочi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    alive: true,
+    get tamagochiStates() {
+        return `Здоров'я  в ${this.name} - ${this.healthLevels} Щастя - ${this.happyLevels} Ситість - ${this.foodLevels}`;
+    },
+    get doGameOver() {
+        return 'Game over!';
+    },
+    eat() {
+        if (!this.alive) {
+            this.doGameOver();
+        }
```

Comment:
You forget to exit the function if the pet is not alive. Please, fix it. Review the whole code where you use it

```suggestion
        if (!this.alive) {
            this.doGameOver();
            return this;
        }
```

or you can use another way to do it ;)